o8g/Sets/Core Set/set.xml

```diff
@@ -130,11 +130,11 @@
       <property name="Type" value="Stronghold"/>
       <property name="Unique" value="Unique"/>
       <property name="Traits" value="City."/>
-      <property name="Text" value="???"/>
```

Comment:
Hmm I'm not sure why these changes are showing up, they should already exist and don't need to be merged